requirements.txt

```diff
@@ -0,0 +1,10 @@
+boto3==1.13.16
+botocore==1.16.16
+docutils==0.15.2
+jmespath==0.10.0
+python-dateutil==2.8.1
+s3transfer==0.3.3
+six==1.15.0
+urllib3==1.25.9
+flake8==3.8.3
```

Comment:
you should consider breaking this into requirements.txt and requirements-dev.txt where requirements-dev have development only deps which is the popular convention.

Reply:
let me see if there are any dependencies other than I can see the docutils and six or others that are only specific to development.